recipe-server/client/control/routes.js

```diff
@@ -27,8 +27,10 @@ const routes = {
         crumb: 'Recipes Listing',
       },
       '/new': {
-        component: CreateRecipePage,
-        crumb: 'New Recipe',
+        '/': {
+          component: CreateRecipePage,
+          crumb: 'New Recipe',
+        }
```

Comment:
I hit approve as I noticed this is missing a comma. I imagine CI will fail due to lint